core_apps/acme/templates/services/app/cm.yml.j2

```
@@ -0,0 +1,15 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: letsencrypt
+  annotations:
+    "acme.openshift.io/priority": {% if acme_env == "live" %}"100"{% else %}"50"{% endif %}
```

I don't think that this test is needed, since the priority annotation is only used to [sort a list](https://github.com/tnozicka/openshift-acme/blob/8b54945f9457b1b960fea133bce359679eee5d62/pkg/controllerutils/controllerutils.go#L51-L89) if multiple CertIssuer are defined.

```suggestion
    "acme.openshift.io/priority": 100
```

---

True.